stdlib/public/core/ImplicitlyUnwrappedOptional.swift

```diff
@@ -73,12 +73,11 @@ public enum ImplicitlyUnwrappedOptional<Wrapped>
   }
 
   /// Returns a mirror that reflects `self`.
-  public func _getMirror() -> _MirrorType {
-    // FIXME: This should probably use _OptionalMirror in both cases.
+  public func customMirror() -> Mirror {
```

Comment:
I'll investigate, but I think that you are right; there are probably a few custom mirrors that don't need to exist. I'll do an audit and see what else is redundant.